controllers/usersController.js

```diff
@@ -15,14 +15,14 @@ function create(req, res) {
   });
 }
 
-// GET /api/users/:userId
```

Comment:
In your `server.js`, your route is actually `/api/users/:user_id` (throughout this file).